### PR TITLE
1984 Deathsquads

### PR DESCRIPTION
**Pull request by Unkn0wnGh0st333**
## About the PR
Comments out every item involving Deathsquad and remove mentions of Deathsquad.
Edits the Centcomm map removing Deathsquad gear and adds ERT gear as replacement.

## Why / Balance
Deathsquad does not work with the direction we want to go in the future. It will never be used event-wise and provides absolutely nothing with its existence besides being a "joke" when someone has the chameleon gear. My solution? Full comment it out. 

## Technical details
Comments out:
- Code Epsilon
- Jumpsuit
- Gas Mask
- Hardsuit + Helmet
- PDA + ID
- Spawner and Humanoid
- Pulse Rifle (DS variant)
- Job + Time Tracker

Centcomm changes:
- Removes DS armor and replaces it with ERT Security
- Adds a new ERT leader room
- Uses the two unused vaults to make two new ERT rooms

## Media
image
image
image
image

## Requirements
- [X] I have tested all added content and changes.
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes
9cxm9c

**Changelog**
🆑

- Deathsquads are and were never real. Thank you for working at Nanotrasen.

### CODE REVIEW

**Review by MilonPL — CHANGES_REQUESTED** [review]
This should be migrated rather than removing it manually from the map. Don't comment entities out, just put them in the HideSpawnMenu category.

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/Revolvers/revolvers.yml
Comment (StarboundSolaris) [inline Resources/Prototypes/Entities/Objects/Weapons/Guns/Revolvers/revolvers.yml:1]: This is just a request from me, but If possible, could this be changed to "The Iconic sidearm of the Nanotrasen ERT. Packs a powerful punch. Uses .45 magnum ammo."

**Comment by Monotheonist** [comment]
If there's any way you could make the DS gear blacklisted from Chameleon stuff (unless this already does it,) please do so. What comes to mind is the couple times I've seen people try to RP being Deathsquad in character when they wouldn't even know about what it is IC which is always a moment and a half, usually.

Also I'm pretty sure Epsilon's useful for the like 1 scenario where it might be needed, but also it'd have to like. Be recontextualized, so, I dunno.

**Comment by Radezolid** [comment]
Why remove it completely? Just remove it from CC and blacklist from chameleon. Leave the other stuff for admemes and what not.

**Comment by Monotheonist** [comment]
> Why remove it completely? Just remove it from CC and blacklist from chameleon. Leave the other stuff for admemes and what not.

Best solution is to remove it from spawning, CC, and Chameleon, there's a *very slim chance* I can see it being used for an admeme because of what Deathsquad means and how much setup is required IMO. Should still stay in the game (since it's an upstream thing), just migrate it out locally here, like Milon said

File: Resources/Prototypes/AlertLevels/alert_levels.yml [hunk 1]
```diff
@@ -61,12 +61,13 @@
...
+    # DeltaV - 1984 Deathsquad
+    #epsilon:
+      #announcement: alert-level-epsilon-announcement
+      #selectable: false
+      #sound: /Audio/Announcements/Alerts/code_epsilon.ogg
+      #disableSelection: true
+      #color: DarkViolet
+      #emergencyLightColor: DarkViolet
+      #forceEnableEmergencyLights: true
+      #shuttleTime: 1200
```
Comment (MilonPL) [inline Resources/Prototypes/AlertLevels/alert_levels.yml:73]: leave it

File: Resources/Prototypes/DeltaV/Entities/Objects/Weapons/Guns/Rifles/rifles.yml [hunk 2]
```diff
@@ -211,4 +211,4 @@
     graph: WeaponRifleMusketGraph
     node: musket
   - type: UseDelay
-    delay: 0 #Straight up doesn't work for guns
\ No newline at end of file
+    delay: 0 #Straight up doesn't work for guns
```
Comment (MilonPL) [inline Resources/Prototypes/DeltaV/Entities/Objects/Weapons/Guns/Rifles/rifles.yml:214]: untouch

File: Resources/Prototypes/DeltaV/Entities/Objects/Weapons/Guns/Rifles/rifles.yml [hunk 3]
```diff
@@ -60,7 +60,7 @@
   parent: [BaseWeaponRifle, BaseGunWieldable]
   id: WeaponRifleJackdaw
   description: The beginning of the end is heralded by the song of a Jackdaw. Uses .25 caseless ammo.
-  suffix: Deathsquad
+  suffix: Admeme #1984 Deathsquad
```
Comment (MilonPL) [inline Resources/Prototypes/DeltaV/Entities/Objects/Weapons/Guns/Rifles/rifles.yml:63]: just remove the suffix

File: Resources/Prototypes/Entities/Structures/Lighting/strobe_lighting.yml [hunk 4]
```diff
@@ -138,13 +138,14 @@
...
+#- type: entity
+  #id: PoweredStrobeLightEpsilon
+  #suffix: "Empty, epsilon"
+  #parent: PoweredStrobeLightEmpty
+  #components:
+  #- type: AmbientSound
+    #volume: 0
+    #range: 10
+    #sound:
+      #path: "/Audio/Effects/Lightning/strobeepsilon.ogg"
```
Comment (MilonPL) [inline Resources/Prototypes/Entities/Structures/Lighting/strobe_lighting.yml:151]: leave

**Comment by Avalon-Proto** [comment]
I'd rather we keep the DS stuff, so on, for other things, than remove it completely. There are things it can be used for, IE a task force meant for dealing with a rogue AI or so on, instead of just contracts terminated, exterminate. Removing from CC, chameleon so on is fine, but I still want the ability to make use of it for other things

File: Resources/Prototypes/Roles/Jobs/CentComm/deathsquad.yml [hunk 5]
```diff
@@ -1,44 +1,45 @@
...
+    #- SpeedLoaderMagnumAP
+    #- SpeedLoaderMagnumAP
+    #- BoxFlashbang
+    #- ToolDebug # spanish army knife
+    #- WelderExperimental
+    #- Hypospray
+    #- DeathAcidifierImplanter # crew will try to steal their amazing hardsuits
+    #- FreedomImplanter
+  #inhand:
+  #- WeaponPulseRifle
```
Comment (MilonPL) [inline Resources/Prototypes/Roles/Jobs/CentComm/deathsquad.yml:45]: no reason to remove it

File: Resources/Prototypes/Roles/Jobs/departments.yml [hunk 6]
```diff
@@ -51,7 +51,7 @@
   - ERTMedical
   - ERTSecurity
   - ERTEngineer
-  - DeathSquad
+  #- DeathSquad # DeltaV - 1984 Deathsquad
```
Comment (MilonPL) [inline Resources/Prototypes/Roles/Jobs/departments.yml:54]: uncomment

File: Resources/Prototypes/Roles/Jobs/departments.yml [hunk 7]
```diff
@@ -77,7 +77,7 @@
...
```
Comment (MilonPL) [inline Resources/Prototypes/Roles/Jobs/departments.yml:80]: uncomment

File: Resources/Prototypes/Roles/play_time_trackers.yml [hunk 8]
```diff
@@ -64,8 +64,9 @@
 - type: playTimeTracker
   id: JobERTSecurity
 
-- type: playTimeTracker
-  id: JobDeathSquad
+# DeltaV - 1984 Deathsquad
+#- type: playTimeTracker
+  #id: JobDeathSquad
```
Comment (MilonPL) [inline Resources/Prototypes/Roles/play_time_trackers.yml:69]: uncomment

**Review by MilonPL — CHANGES_REQUESTED** [review]
still needs a migration instead of modifying the map

**Comment by Monotheonist** [comment]
> I'd rather we keep the DS stuff, so on, for other things, than remove it completely. There are things it can be used for, IE a task force meant for dealing with a rogue AI or so on, instead of just contracts terminated, exterminate. Removing from CC, chameleon so on is fine, but I still want the ability to make use of it for other things

Valid, but the name "deathsquad" inherently implicates that they're the "contracts terminated, exterminate" force, and I dunno if you wanna go through renaming the hardsuits + other gear like 20 times with admin tools.

**Comment by Radezolid** [comment]
> > I'd rather we keep the DS stuff, so on, for other things, than remove it completely. There are things it can be used for, IE a task force meant for dealing with a rogue AI or so on, instead of just contracts terminated, exterminate. Removing from CC, chameleon so on is fine, but I still want the ability to make use of it for other things
> 
> Valid, but the name "deathsquad" inherently implicates that they're the "contracts terminated, exterminate" force, and I dunno if you wanna go through renaming the hardsuits + other gear like 20 times with admin tools.

Or we can just keep it for less upstream merge conflict and make our own version in DeltaV folders for when such a force is needed.
We just need to remove it from the crew's eyes, we don't need to nuke the YMLs for something they can't even see without admin intervention.

**Pull request closed without merging**